content/en/developers/integrations/log_integration.md

```diff
@@ -109,8 +107,16 @@ To easily navigate the facet list, facets are grouped together. For fields speci
 4. In the **Group** field, enter the name and a description of the new group, and select **New group**.
 5. Click **Update**.
 
+**Guidelines**
+- Before creating a new facet for an integration, review if the attribute should be remapped to a [standard attribute](6) instead. Facets for standard attributes will be added automatically by Datadog when the log pipeline is published.
```

Comment:
```suggestion
- Before creating a new facet for an integration, review if the attribute should be remapped to a [standard attribute][6] instead. Facets for standard attributes are added automatically by Datadog when the log pipeline is published.
```

Reply:
@aliciascott  `[standard attribute][6]` would break the link, not sure if it's intentional

Reply:
Hi @LucasC , that is correct, links have to be noted with `[name of link][number not used yet] with the link at the bottom of the page like :

[number not used yet]: the link to the standard attribute page

See: https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1182958054/Markdown#Links

Right now when I check out the branch locally, it 404's going to that link. Hope that helps !

Reply:
Duh, I was confused with the markdown syntax for direct links. Updated!